specs/core/vocab/core-vocab.ttl

```diff
@@ -745,3 +772,14 @@ oslc:executes
   rdfs:isDefinedBy oslc: ;
   rdfs:label "executes" ;
   rdfs:comment "Link from a currently available action to the future action it realizes." .
+
+oslc:publisher
+  a rdf:property ;
+  rdfs: rdfs:isDefinedBy oslc: ;
```

Comment:
what's this?

Reply:
See issue #21, this is the proposed fix to deal with implementations that used oslc:publisher (which may have been used in OSLC 1.0) instead of dcterms:publisher. I was a little reluctant to add this but the issue seemed to indicate we agreed to it.

Reply:
No, I am talking about the RDF statement, which to me makes no sense:

`oslc:publisher rdfs: rdfs:isDefinedBy oslc: .`

I think you meant to write `oslc:publisher rdfs:isDefinedBy oslc: .`

Reply:
Sorry, too many distractions and not enough time. Didn't check it carefully enough.